include/networkit/distance/SSSP.hpp

```diff
@@ -151,7 +151,7 @@ class SSSP : public Algorithm {
      */
     count getSumOfDistances() const {
         assureFinished();
-        return sumDist;
+        return static_cast<count>(sumDist);
```

Comment:
This would work only on unweighted graphs, for weighted graph we need `sumDist` to be a double. Actually, the method should return a `double`, not a `count`.